hutch_python/tests/test_cli.py

```diff
@@ -160,7 +160,8 @@ def __dir__(self):
     # us.
     hutch_python.cli.configure_ipython_session(HutchPythonArgs())
 
-    completer = IPython.core.completer.Completer(namespace=ns)
+    completer = IPython.core.completer.IPCompleter(namespace=ns)
```

Comment:
Is there a secret minimum pin lurking in this rename?

Reply:
If there is I suppose it'd be hard to find

Reply:
Git blame for the `IPCompleter` dates back 18 years, to before the git history started
![image](https://github.com/user-attachments/assets/423cde66-5634-4453-b139-b37b24574135)